nmmo/task/sampler.py

```diff
@@ -0,0 +1,44 @@
+import random
+
+from nmmo.task.predicate import Predicate
+from nmmo.task.predicate.core import AND, NOT, OR
+
+class RandomTaskSampler:
```

Comment:
Good, this is important

Reply:
This is technically a generator, not a sampler though

Reply:
Should we move this into baselines?